### PR TITLE
[NEW PROVIDER] Support to deploy Cloud Foundry in Alibaba Cloud

### PR DESCRIPTION
**Pull request by xiaozhu36**
The PR will support to deploy Cloud Foundry in Alibaba Cloud. It has one directory 'iaas-support/alicloud' that contains cloud-config and some scripts that can help user to deploy cloud foundry in China region.

### CODE REVIEW

File: iaas-support/alicloud/download-releases.sh [hunk 1]
```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
```
Comment (cppforlife) [inline iaas-support/alicloud/download-releases.sh]: why do we need these two scripts?

Comment (xiaozhu36) [reply iaas-support/alicloud/download-releases.sh]: These two scripts aim to resolve downloading cf releases failed in China mainland results from bad network or other uncertain reasons.

**Comment by dsabeti** [comment]
Hi @xiaozhu36, it looks like you don't publicize your membership in the org that signed the CLA. You can follow instructions to do so here: https://help.github.com/articles/publicizing-or-hiding-organization-membership

**Comment by xiaozhu36** [comment]
Hi @dsabeti is it required for finishing all of CLA progress?

**Comment by dsabeti** [comment]
Hi @xiaozhu36. Yes, publicizing your github org membership is the way our automation validates that you are part of an organization that has signed the CLA.

**Comment by xiaozhu36** [comment]
Hi @dsabeti I have publicized my github org membership and reopened the PR, but the PR still required me to signed the CLA. Could you give me some help? thanks a lot.

**Comment by dsabeti** [comment]
Hi @xiaozhu36, we finally have the CLA verified, so I'll prioritize the story to merge your PR.

**Comment by xiaozhu36** [comment]
Hi @dsabeti Thanks for your help. Do I need to reopen other PRs? like [bosh-deployment](https://github.com/cloudfoundry/bosh-deployment/pull/175), [docs-bosh](https://github.com/cloudfoundry/docs-bosh/pull/429).

**Comment by dsabeti** [comment]
Yes, you'll need to re-open those PRs.

**Comment by xiaozhu36** [comment]
HI @dsabeti Thanks a lot. Got it.

**Pull request merged**